conans/model/conan_file.py

```diff
@@ -278,8 +278,8 @@ def recipe_metadata_folder(self):
         return self.folders.recipe_metadata_folder
 
     @property
-    def pkg_metadata_folder(self):
-        return self.folders.pkg_metadata_folder
+    def package_metadata_folder(self):
```

Comment:
Just a rename, to not use the ``pkg`` short version, everything else UI in Conan is ``package``.